ci/long_running_tests/workloads/many_actor_tasks.py

```diff
@@ -45,8 +45,10 @@ def method(self):
         self.value += 1
 
 
+# TODO (williamma12): Update the actors to each have only 0.1 of a cpu once
+# https://github.com/ray-project/ray/issues/4312 gets resolved.
 actors = [
-    Actor._remote([], {}, num_cpus=0.1, resources={str(i % num_nodes): 0.1})
+    Actor._remote([], {}, num_cpus=1, resources={str(i % num_nodes): 0.1})
```

Comment:
I don't think this will work because we're creating `num_nodes * 5` actors, each requiring 1 CPU, but each node has only 2 CPUs.

Reply:
Ah ok, I increased the number of cpus per node to 10. I think that should work now?

Reply:
Did you try it out and see if it runs?